DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -181,14 +181,20 @@ public List<DelegateUser> GetDelegateUsersByEmailAddress(string emailAddress)
         {
             List<DelegateUser> users = connection.Query<DelegateUser>(
                 @"SELECT
-                        CandidateID,
-                        FirstName,
-                        LastName,
-                        EmailAddress,
-                        Password,
-                        ResetPasswordID
-                    FROM Candidates
-                    WHERE (EmailAddress = @emailAddress)",
+                        cd.CandidateID AS Id,
+                        cd.CandidateNumber,
+                        cd.CentreID,
+                        ct.CentreName,
+                        ct.Active AS CentreActive,
+                        cd.EmailAddress,
+                        cd.FirstName,
+                        cd.LastName,
+                        cd.Password,
+                        cd.Approved,
+                        cd.ResetPasswordID

```

Comment:
Do we need to change any test to check that these all come through fine now?

Reply:
There are actually no tests for the GetByEmail methods. Should I write some as part of this change?